components/salsa-2022/src/accumulator.rs

```diff
@@ -148,6 +148,14 @@ where
     fn salsa_struct_deleted(&self, _db: &DB, _id: crate::Id) {
         panic!("unexpected call: accumulator is not registered as a dependent fn");
     }
+
+    fn fmt_index(
+        &self,
+        _index: Option<crate::Id>,
+        _fmt: &mut std::fmt::Formatter<'_>,
+    ) -> std::fmt::Result {
+        todo!()
```

Comment:
Ideally, we'd give a debug name to accumulators, and then do something like

```rust
if let Some(i) = index {
    write!(fmt, "{}({})", self.debug_name, u32::from(i))
} else {
    write!(fmt, "{}()", self.debug_name)
}
```

(Actually, given how often this little snippet will occur across various ingredients, maybe we should add a helper function to do it.)